content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -17,13 +17,154 @@ aliases:
 
 ## Lassie
 
-Lassie is a simple retrieval client for Filecoin. It finds and fetches your data over the best retrieval protocols available.
+Lassie is a simple retrieval client for Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command:
 
-### Prerequisites
+```shell
+lassie fetch <CID>
+```
+
+Lassie can also be used as a reference library to fetch data from Filecoin. In the absence of a single uber-protocol which meet all data transfer needs, Lassie was designed to use multiple protocols for data transfers.
+
+![Lassie Architecture](lassie_architecture.png)
+
+### Lassie HTTP daemon
+
+The Lassie HTTP Daemon is an HTTP interface for retrieving IPLD data from IPFS and Filecoin peers. It fetches content over the GraphSync and Bitswap protocols and provides the resulting data in CAR format.
+
+```shell
+GET /ipfs/{cid}[/path][?params]
+```
+
+Retrieves from peers that have the content identified by the given root CID, streaming the DAG in the response in [CAR (v1)](https://ipld.io/specs/transport/car/carv1/) format.
+
+#### Request
```

Comment:
Same as my comment on the CAR format section, there's too much detail here, it's going to change a lot over time as we develop further, and the daemon is not a standard usage mode we imagine a lot of people using. They can be pointed directly to https://github.com/filecoin-project/lassie/blob/main/docs/HTTP_SPEC.md for all this info. A summary if anything would do I think.